native-pkcs11-keychain/src/macos/key.rs

```diff
@@ -57,6 +57,18 @@ fn sigalg_to_seckeyalg(
         native_pkcs11_traits::SignatureAlgorithm::RsaPkcs1v15Sha512 => {
             RSASignatureMessagePKCS1v15SHA512
         }
+        native_pkcs11_traits::SignatureAlgorithm::RsaPss {
+            digest: _digest,
+            mask_generation_function,
+            salt_length: _,
+        } => match mask_generation_function {
+            //  Assume digest == mgf and salt_length == len(digest).
```

Comment:
are these safe assumptions?

Reply:
macos doesn't allow any other parameter combination, but we could definitely throw an error instead of depending on SecurityFramework to fail gracefully.

Reply:
Latest commit checks this and returns an Error if it is not true